seatunnel-formats/seatunnel-format-text/src/main/java/org/apache/seatunnel/format/text/TextSerializationSchema.java

```diff
@@ -134,11 +136,18 @@ private String convert(Object field, SeaTunnelDataType<?> fieldType, int level)
             case INT:
             case STRING:
             case BOOLEAN:
-            case TINYINT:
             case SMALLINT:
             case BIGINT:
             case DECIMAL:
                 return field.toString();
+            case TINYINT:
```

Comment:
What the meaning of this? Under what circumstances will tinyint toString lose accuracy?

Reply:
Since in ST, TINYINT will store as Byte.
If we use `toString` here, the Byte will be transform to Int.
```
public static String toString(byte b) {
        return Integer.toString((int)b, 10);
}
```
This is not byte, will throw exception in deserialization.

Reply:
This is used for compatibility with `UNSIGNED_TINYINT`, since in ST, `UNSIGNED_TINYINT` might be transform to Short(e.g. MysqlDataTypeConvertor), so the value here is not alway byte, and we need to transform the value into byte if it is short, otherwise the type is incorrect and we will get exception in deserialization.

Reply:
Maybe use `TINYINT` to match the `UNSIGNED_TINYINT ` is incorrect, I will close this pr.